src/microsoft/office/license/user/microsoft-office-license-user.controller.js

```diff
@@ -1,73 +1,50 @@
 angular.module("Module.microsoft.controllers").controller("MicrosoftOfficeLicenseUsersCtrl", class MicrosoftOfficeLicenseUsersCtrl {
 
-    constructor (MicrosoftOfficeLicenseService, $stateParams, $scope, $timeout, Alerter) {
+    constructor (MicrosoftOfficeLicenseService, $stateParams, $scope, $timeout, Alerter, translator) {
         this.license = MicrosoftOfficeLicenseService;
         this.$stateParams = $stateParams;
+        this.currentLicense = $stateParams.serviceName;
```

Comment:
Could you please set this in the `$onInit` ? We're trying to have only service initialization in the `constructor` :smiley:

Reply:
This is making more sense as is, inside constructor. @FredericEspiau look more closely, it's an initialization ```currentLicense``` is assign with another var, not a function. Since this ```$stateParams.serviceName``` will not change between instantiation and $oninit you could remove this.$stateParams assignation.